test/lib/send_with_us/api_request_test.rb

```diff
@@ -103,6 +103,25 @@ def test_send_with_headers
     assert_instance_of( Net::HTTPOK, result )
   end
 
+  def test_send_with_tags
+    build_objects
+    email_id = 'tem_9YvYsaLW2Mw4tmPiLcVvpC'
+    result = @api.send_with(
+      email_id,
+      {name: 'Ruby Unit Test', address: 'matt@example.com'},
+      {name: 'sendwithus', address: 'matt@example.com'},
+      {},
+      [],
+      [],
+      [],
+      '',
+      'v2',
+      {},
+      ['tag1', 'tag2']
```

Comment:
wow, this function has way too many params :cry:. I would not want to use this client.